src/test/regress/sql/misc_jiras.sql

```diff
@@ -70,3 +70,8 @@ from gp_dist_random('gp_distribution_policy') where localoid = 't2_17271'::regcl
 drop table t_17271;
 drop table t1_17271;
 drop table t2_17271;
+
+-- Github Issue 1055
+-- AO column storage not supported for zero-column tables
+set default_table_access_method = ao_column;
```

Comment:
Please add more test cases here:
1. `alter table ... drop column ...` to drop columns one by one until to zero-column.
2. `alter table ... set access method` from other type of table storage to aocs.

Reply:
> drop columns one by one until to zero-column

wild to reject drop column to zero ?